synapse/storage/end_to_end_keys.py

```diff
@@ -79,6 +80,7 @@ def get_e2e_device_keys(
             include_deleted_devices (bool): whether to include null entries for
                 devices which no longer exist (but were in the query_list).
                 This option only takes effect if include_all_devices is true.
+            req_user_id: The user requesting the device list
```

Comment:
again, please can this have a type, an explanation of what it is used for, and an explanation of what `None` means.